StencilSwiftKit.xcodeproj/project.pbxproj

```diff
@@ -205,6 +211,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
 				en,
```

Comment:
We should probably remove `en` if the new official value to specify that locale is supposed to be `English`. Or should that be the other way around?